tensorboard/webapp/theme/_tb_theme.template.scss

```diff
@@ -36,11 +36,13 @@ $tb-foreground: map_merge(
     secondary-text: mat-color($mat-gray, 700),
     disabled-text: mat-color($mat-gray, 600),
     // TB specific variable.
-      border: #ebebeb,
+    border: #ebebeb,
     link: mat-color($mat-blue, 700),
   )
 );
 
+$tb-background: $mat-light-theme-background;
```

Comment:
Can this be selected directly from $tb-theme? 

e.g. replace this with a line on L52
`$tb-background: map-get($tb-theme, background);`